packages/table/src/cell/formats/truncatedFormat.tsx

```diff
@@ -92,6 +97,18 @@ export class TruncatedFormat extends React.Component<ITruncatedFormatProps, ITru
         truncationSuffix: "...",
     };
 
+    // these default values for `resizeRowsByApproximateHeight` have been
+    // fine-tuned to work well with default Table font styles.
+    private static resizeRowsByApproximateHeightDefaults: Record<
+        keyof IResizeRowsByApproximateHeightOptions,
+        number
+    > = {
+        getApproximateCharWidth: 8,
+        getApproximateLineHeight: 18,
+        getCellHorizontalPadding: 2 * Locator.CELL_HORIZONTAL_PADDING,
```

Comment:
Kinda weird to import Locator just to grab the constants within it. Can we pull these constants out into a separate `tableConstants.ts` file?

Reply:
...I mean, that in and of itself seems somewhat strange to me. Should we put other constants in there too? Which?